### PR TITLE
Fixes #173 Fixed dropdown menus

### LINKED ISSUE #173: Fix Dropdown menu structure and views across apps and pages
Related to: https://github.com/fossasia/chat.susi.ai/issues/657

**Not logged in users:**
About -> Overview
Chat -> /chat app
Skills -> / skills app
Settings -> /settings
Login -> login

**Logged in users:**
About -> Overview
Chat -> /chat app
Skills -> / skills app
Settings -> /settings
[Themes -> theme popup (only in chat app)]
Logout -> logout

* Add icons to all menu items
* Don't show "signup" on menu. The link to signup on the login page is sufficient.
* On chat app direct settings to chat app settings
* On skills cms direct settings link to skills settings
* Ensure the dropdown menu works on the same way on chat pages overview etc.

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes #173  Dropdown menu structure

Changes: Added proper menu structure

Link to deployment for testing: http://susiskillmadhav.surge.sh/

Screenshots for the change: 
cms2
(Logged in)

csm1

(Logged out)

### CODE REVIEW

**Comment by madhavrathi** [comment]
@dynamitechetan @rishiraj824 @saurabhjn76 @DravitLochan Please Review
Thanks

**Comment by mariobehling** [comment]
Skill Editor should not be included in the dropdown. The Skill editor should be include as a + sign on the top right of the skill overview page.

**Comment by mariobehling** [comment]
@madhavrathi Please follow up here on the Add skill + featured https://github.com/fossasia/susi_skill_cms/issues/157

File: src/components/StaticAppBar/StaticAppBar.react.js [hunk 1]
```diff
@@ -11,29 +11,43 @@ import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
...
+        onTouchTap={this.handleClose}
+        containerElement={<Link to="/overview" />}
+        rightIcon={<Info/>}/>
+      <MenuItem
+        href="http://chat.susi.ai/"
+        rightIcon={<Chat/>}>
+        Chat
+      </MenuItem>
+      <MenuItem
+        href="http://skills.susi.ai/"
```
Comment (uday96) [inline src/components/StaticAppBar/StaticAppBar.react.js]: @madhavrathi This should be redirecting to '/' ?

Comment (madhavrathi) [inline src/components/StaticAppBar/StaticAppBar.react.js]: Yes, good point we can save the reloading here. I will add this in the update.
Thanks

**Comment by madhavrathi** [comment]
@mariobehling I have updated the PR accordingly. Please Review
Thanks

**Comment by mariobehling** [comment]
Please move the plus button to the right of the below white area instead of the menu bar.
screenshot from 2017-08-01 22-54-13

**Comment by madhavrathi** [comment]
@mariobehling Please review: http://susiskillmadhav.surge.sh/ I have made the changes.

**Comment by dynamitechetan** [comment]
@mariobehling @madhavrathi I have a suggestion for the fab button position, fixed at the bottom right of the page.
image

**Comment by mariobehling** [comment]
@dynamitechetan Good idea. Let's add this as an item for the redesign of the page.

**Pull request merged**